docs/Config.md

```diff
@@ -75,3 +75,45 @@ The secrets in this example are dummy values and won't work in an actual app.
   }
 }
 ```
+
+### Running over HTTPS
+
+You need to change change the `protocol` to `https`, and tell the server where
```

Comment:
That would be helpful but I also wonder if this is something we should address. 

I am lazy. when i type `beta2.meetup.com` i see a broken page. Then i type `https://beta2.meetup.com.` Its kinda annoying.